### ISSUE TITLE
Dropdown: required attribute not set in hidden input element

### ISSUE THREAD

**Issue by mrkshx** [primefaces/primeng#14322]
### Describe the bug

When setting "required" attribute to Dropdown component, the screen reader doesn't inform the user that this dropdown is required as it is only added to p-dropdown element. 

### Environment

Primeng 16

also tested on Primeng 14 and 17

### Reproducer

https://stackblitz.com/edit/2yfb3e?file=src%2Fapp%2Fdemo%2Fdropdown-basic-demo.html

### Angular version

16.2.12

### PrimeNG version

16.9.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

latest

### Steps to reproduce the behavior

- Add required attribute to p-dropdown
- Open screen reader ( I tested with NVDA )
- Tab focus the dropdown
- Screen reader will not inform that dropdown is required

### Expected behavior

required and aria-required attribute should be available for screen reader so that user gets informed that the dropdown element is required.
